toolchain/personae.md

```diff
@@ -0,0 +1,56 @@
+
+As a tech writer...
+
+- I want to use a simple tool to manage docs-as-code so I can concentrate on writing.
+- I want to have access to spell/grammar check so I can take advantage of existing tools.
+- I want to have the option to use the editor of my choice so I don't have to learn something new.
+
+## Developer
+
+Often times **Developers** are called on to document the code or application they've created. In business this may be a matter of convenience, where in open source projects there may simply not be anyone else available. But in either case this is a secondary task added to their main roles, often without enough thought given to the effort required to produce something of quality. These individuals are (obviously) very technically savvy, although this doesn't preclude them from wanting to use GUI tools.
+
+As a developer...
+
+- I want help in deciding what form the required documentation should take so I don't need to learn technical writing theory.
+- I want help to stage the required documentation so I don't need to create everything from scratch.
+
+## Reviewer
+
+**Reviewers** can take many forms, from technical leads and delivery managers responsible for the solution as a whole to subject matter experts who may be getting their first look at the solution they themselves will use. The reviewer's role is to look over the documentation and provide *feedback* to be incorporated back into the material.
+
+As a reviewer...
+
+- I want to look review the documentation in its final presentation so I can better visualize how it's used.
+- I want the ability to easily add comments to it so the review process is easy to track.
+
+As a technical reviewer...
+
+- I want the ability to make corrections directly in the documentation's source so I don't need to go through the review process.
+
+## Marketing
+
+While **Marketing** reviewer are the same as technical reviewers in many ways (and many of the same needs will apply), they are also looking after the company's branding, and therefore may have additional requirements.
+
+As a marketing reviewer...
+
+- I want the ability to comment on the look-and-feel of the deliverables, so I can ensure they're in line with the brand.
+
+## Technology decision-maker
+
+The **Technology Decision Maker** may weigh in on the documentation as a 
+
+As a technology decision maker...
+
+- I want to provide feedback on the approach to the deliverable(s), to ensure it's meeting expectations as part of the overall solution.
+- I want to enforce the docs-as-code approach, to ensure it's in line with my high-level strategy for the product(s).
```

Comment:
```suggestion
- I want to encourage the docs-as-code approach, to ensure it's in line with my high-level strategy for the product.
```
"Enforce" seems a little heavy handed. I'm also not entirely certain this is true. Most managers/POs don't understand or particularly care about docs as code (in my experience).

Reply:
I kinda like this statement -- I think that we should draw a line in the sand from a values-perspective.  That doc-as-code is the future for building scale-able and inclusive teams.

Reply:
I think there are a few more personas worth calling out, which I've been listing in my tech writing pattern presentations:
A developer, with a deep understanding of the software being described.
A product manager, who has a holistic view of the application and it’s roadmap.
A software user, able to explain the application within the context of the application’s domain.
An “expert newbie”. Some of the best feedback comes from new users who get stuck.
An educator, who understands the principles of learning.
An information architect, who understands how to structure material.
A software support person, who learns all the pain points of users.
A researcher, who proactively identifies pain points and market opportunities.
A writer, who writes clearly and concisely with good grammar.
Translators, for translating to target languages.
A DevOps person, who can set up doc build pipelines.
A community builder, and coordinator, who can inspire collective action, capture offers of help, and help all these different personas to collaborate together.